components/did/Core/Inc/LCD.h

```diff
@@ -39,13 +39,13 @@ Header file containing all constants and function declarations to the LCD
 // MCB Drive State
 #define STATE_XPOS 0
 #define STATE_YPOS 6
-#define STATE_DATA_XPOS 20
+#define STATE_DATA_XPOS 24
 #define STATE_DATA_YPOS 6
-// Target Cruise Speed
-#define CRUISE_XPOS 0
-#define CRUISE_YPOS 9
-#define CRUISE_DATA_XPOS 20
-#define CRUISE_DATA_YPOS 9
+// Regen Enabled
+#define REGEN_XPOS 0
+#define REGEN_YPOS 9
+#define REGEN_DATA_XPOS 20
+#define REGEN_DATA_YPOS 9
```

Comment:
So we are replacing the Cruise display bit with regen now is what this looks like. Have we verified that this is accurately displayed on the Car? As in no wrapping AND correctly reading the incoming message?

Reply:
Have to validate on the car. We won't merge until it is validated.